javascript-functions/main.js

```diff
@@ -0,0 +1,39 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  return seconds;
+}
+var convertMinutesToSecondsResult = convertMinutesToSeconds(10);
+console.log('Convert minutes to seconds result:', convertMinutesToSecondsResult);
+
+function greet(name) {
+  var greeting = 'Hey' + ' ' + name + '!';
+  return greeting;
+}
+var greetingResult = greet('Rachel');
+console.log('Greeting Result:', greetingResult);
+
+function getArea(width, height) {
+  var area = width * height;
+  return area;
+}
+var areaResult = getArea(20, 20);
+console.log('Area result:', areaResult);
+
+function getFirstName(person) {
+  var name = {
```

Comment:
You would not be making this value in the function, this should be received as an argument that will fill your `person` parameter.